contribs/gnodev/cmd/gnodev/setup_node.go

```diff
@@ -10,43 +10,57 @@ import (
 	gnodev "github.com/gnolang/gno/contribs/gnodev/pkg/dev"
 	"github.com/gnolang/gno/contribs/gnodev/pkg/emitter"
 	"github.com/gnolang/gno/gno.land/pkg/gnoland"
-	"github.com/gnolang/gno/tm2/pkg/std"
+	"github.com/gnolang/gno/tm2/pkg/bft/types"
 )
 
 // setupDevNode initializes and returns a new DevNode.
 func setupDevNode(
 	ctx context.Context,
-	logger *slog.Logger,
-	cfg *devCfg,
-	remitter emitter.Emitter,
-	balances gnoland.Balances,
-	pkgspath []gnodev.PackagePath,
+	devCfg *devCfg,
+	nodeConfig *gnodev.NodeConfig,
 ) (*gnodev.Node, error) {
-	// Load transactions.
-	txs, err := parseTxs(cfg.txsFile)
-	if err != nil {
-		return nil, fmt.Errorf("unable to load transactions: %w", err)
+	logger := nodeConfig.Logger
+
+	if devCfg.txsFile != "" { // Load txs files
```

Comment:
What happens if I want to have an initial genesis file, but also have an additional transactions one?

Why don't we just load the genesis first, always, and later append the txs from the "transactions sheet"?

Reply:
see my comment:
> Thinking more about this, I believe we should not authorize balance-file and/or txs-file along with genesis-file, as it would make things more complex and is likely not very useful. If we find a use case for this in the 
future, we can revisit and rework this part.

I think it's a hard-to-predict behavior. For now, I prefer to keep things simple. If both of those flags are specified, it will raise an error. If it becomes clear that people need this feature, we can consider adding it later.